lib/temperature_monitor/temperature_handler.ex

```diff
@@ -0,0 +1,91 @@
+defmodule TemperatureMonitor.TemperatureHandler do
+  use GenServer
+
+  def start_link(stash_pid) do
+    {:ok, _pid} = GenServer.start_link(__MODULE__, stash_pid, name: __MODULE__)
+  end
+
+  def init(stash_pid) do
+    filelist = TemperatureMonitor.Stash.get_value stash_pid
+    monitored_paths = []
+     {:ok, {filelist, stash_pid, monitored_paths}}
+  end
+
+  def callback(:stop) do
```

Comment:
do you need these `callback/1` and `callback/2` functions here? I think you just need them in `TemperatureMonitor.Monitor` since that is the only module that is getting registered with `ExFSWatch`